src/cpp/dynamic/AllowedTopicList.cpp

```diff
@@ -35,46 +42,18 @@ void AllowedTopicList::clear()
     throw UnsupportedException("AllowedTopicList::clear not supported yet");
 }
 
-void AllowedTopicList::block_topic(
-        const AbstractTopic& new_topic)
-{
-    // TODO
-    throw UnsupportedException("AllowedTopicList::block_topic not supported yet");
-}
-
-void AllowedTopicList::allow_topic(
-        const AbstractTopic& topic)
-{
-    // TODO
-    throw UnsupportedException("AllowedTopicList::allow_topic not supported yet");
-}
-
 bool AllowedTopicList::is_topic_allowed(
         const RealTopic& topic) const
 {
     // TODO
     throw UnsupportedException("AllowedTopicList::is_topic_allowed not supported yet");
 }
 
-bool AllowedTopicList::are_topics_allowed_by_default() const
-{
-    // TODO
-    throw UnsupportedException("AllowedTopicList::are_topics_allowed_by_default not supported yet");
-}
-
-void AllowedTopicList::allow_topics_by_default(
-        bool status)
-{
-    // TODO
-    throw UnsupportedException("AllowedTopicList::allow_topics_by_default not supported yet");
-}
-
-void AllowedTopicList::reload(
```

Comment:
I don't see how are you planning to add/remove topics to/from the AllowedTopicList without this function